monoio/Cargo.toml

```diff
@@ -75,6 +75,8 @@ mkdirat = []
 unlinkat = []
 # renameat op(requires kernel 5.11+)
 renameat = []
+# symlinkat op(requires kernel 5.11+)
```

Comment:
It seems require 5.15+ ?

Reply:
https://elixir.bootlin.com/linux/v5.15.1/A/ident/IORING_OP_MKDIRAT

Reply:
> It seems require 5.15+ ?

Yes, I've changed this to 5.15+.